.pre-commit-config.yaml

```
@@ -40,16 +40,12 @@ repos:
       - id: check-executables-have-shebangs
       - id: check-shebang-scripts-are-executable
       - id: check-docstring-first
-      - id: debug-statements
       - id: destroyed-symlinks
```

```suggestion
      - id: debug-statements
      - id: destroyed-symlinks
```
Can you add the first one back? 😅